config/envs/default.py

```diff
@@ -94,20 +94,13 @@ class DefaultConfig:
     APPLICATION_SEARCH_ENDPOINT = CommonConfig.APPLICATION_SEARCH_ENDPOINT
 
     # Assessment store endpoints
-    ASSESSMENTS_STATS_ENDPOINT = (
+    ASSESSMENTS_STATS_FLAGS_ENDPOINT = (
```

Comment:
can we name it to `APPLICATION_OVERVIEW_ENDPOINT_FUND_ROUND_PARAMS` like before ? this is not a flag specific endpoint right

Reply:
of course, good shout